### PR TITLE
Create 2015-09-24-newly-redesigned-developer-portal

### PR DESCRIPTION
**Pull request by trentgillaspie**
Blog post updating visitors on the latest changes to the DRC blog. Has been reviewed by Jesse and Anne.

### CODE REVIEW

File: _posts/2015-09-24-newly-redesigned-developer-portal [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
++You may have noticed our Developer Portal looks different. Our Developer Portal provides you with everything you need to build powerful, scalable apps using our Software Developer Kits (SDK) built on open-source platforms and our extensive APIs. We have redesigned Developer.Rackspace.com.
++
++<!-- more -->
++
++The new design creates a consistent experience for you to interact with our updated developer guides and API reference content (just check out our new [Rackspace Cloud Guide to Core Infrastructure Services](https://developer.rackspace.com/docs/user-guides/infrastructure/).
++
++This is the first step in rolling out an enhanced customer experience across all of our Product and Support documentation. The newly designed Developer Portal was designed with the customer in mind. We’ve worked with our User Research team and taken feedback from you in order to align directly with your expectations in the updated design. So, take a look around and let us know what you think by [providing feedback to us here](https://feedback.rackspace.com/forums/319947-developer-rackspace-com).  
++
++Happy Developing!
++
```
Comment (smashwilson) [inline _posts/2015-09-24-newly-redesigned-developer-portal]: It looks like you have `+`s at the [beginning of each line](https://raw.githubusercontent.com/trentgillaspie/docs-developer-blog/patch-1/_posts/2015-09-24-newly-redesigned-developer-portal) in your source?

**Comment by smashwilson** [comment]
@catlook: Care to work your magic, when you get the chance? :smile:

**Comment by catlook** [comment]
Will do!


-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting - the #1 Managed Cloud Company
DevDocs on Developer Experience Team
Office: (210) 312-8193   Cell: (210) 364-1060
cat.lookabaugh@rackspace.com

We should not write so that it is possible for the reader to understand us, but so that it is impossible for him to misunderstand us. - Quintilian, rhetorician (c. 35-100)

Harmony + Learner + Responsibility + Context + Intellection + Arranger + Consistency + Relator + Input + Analytical

File: _posts/2015-09-24-newly-redesigned-developer-portal [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
++date: 2015-09-24
++comments: true
++author: Trent Gillaspie
++published: true
++categories:
++    - rackspace
++    - announcements
++---
++
++You may have noticed our Developer Portal looks different. Our Developer Portal provides you with everything you need to build powerful, scalable apps using our Software Developer Kits (SDK) built on open-source platforms and our extensive APIs. We have redesigned Developer.Rackspace.com.
```
Comment (catlook) [inline _posts/2015-09-24-newly-redesigned-developer-portal]: You may have noticed our Developer Portal looks different. This portal provides you with everything you need to build powerful, scalable apps using our Software Developer Kits (SDK), which are built on open-source platforms and take advantage of our extensive APIs. We have redesigned Developer.Rackspace.com.

File: _posts/2015-09-24-newly-redesigned-developer-portal [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
++
++<!-- more -->
++
++The new design creates a consistent experience for you to interact with our updated developer guides and API reference content (just check out our new [Rackspace Cloud Guide to Core Infrastructure Services](https://developer.rackspace.com/docs/user-guides/infrastructure/).
++
++This is the first step in rolling out an enhanced customer experience across all of our Product and Support documentation. The newly designed Developer Portal was designed with the customer in mind. We’ve worked with our User Research team and taken feedback from you in order to align directly with your expectations in the updated design. So, take a look around and let us know what you think by [providing feedback to us here](https://feedback.rackspace.com/forums/319947-developer-rackspace-com).
```
Comment (catlook) [inline _posts/2015-09-24-newly-redesigned-developer-portal]: ... newly designed Developer Portal was crafted with the customer ...

File: _posts/2015-09-24-newly-redesigned-developer-portal (hunk 1 above)
Comment (catlook) [inline _posts/2015-09-24-newly-redesigned-developer-portal]: [ ed note: use edits you like and toss the rest!  Thanks :) - Cat ]

**Pull request merged**